CPT4/load_stage.sql

```diff
@@ -635,53 +436,126 @@ FROM (
 		AND s.atv IS NOT NULL
 	GROUP BY concept_code
 	) i
-WHERE i.concept_code = cs.concept_code;
+WHERE i.concept_code = cs.concept_code; -- 29
+
+--15. Update domain_id according to the source OR devv5
```

Comment:
I started the review here
https://docs.google.com/spreadsheets/d/1zIoXHVHDd9JqV9XMQWgbRnULegxogZZUmuTpsg9gVlY/edit#gid=0